pageserver/src/http/routes.rs

```diff
@@ -567,36 +521,32 @@ async fn tenant_status(request: Request<Body>) -> Result<Response<Body>, ApiErro
             false
         });
 
-    let tenant_state = match tenant {
-        Ok(tenant) => tenant.current_state(),
+    let (tenant_state, current_physical_size) = match tenant {
+        Ok(tenant) => {
+            let timelines = tenant.list_timelines();
+            let current_physical_size = tokio::task::spawn_blocking(move || {
```

Comment:
Looks like we can omit this `spawn_blocking` entirely? I see no blocking operations in the `get_physical_size` method (prometheus metrics are based on atomics).

https://github.com/neondatabase/neon/blob/e5e40a31f4ce84a37f88aad2fe612d144fc55e6f/pageserver/src/tenant/timeline.rs#L386-L388

Reply:
Fixed, thanks!